docs/user/alerting/alerting-production-considerations.asciidoc

```diff
@@ -1,28 +1,30 @@
 [role="xpack"]
-[[alerting-scale-performance]]
-== Scale and performance
+[[alerting-production-considerations]]
+== Production considerations
 
-{kib} alerting run both alert checks and actions as persistent background tasks. This has two major benefits:
+{kib} alerting run both alert checks and actions as persistent background tasks managed by the Task Manager. This has two major benefits:
```

Comment:
Is it worth explcitly saying "Kibana Task Manager"?

I think ES has a TM (or tasks of their own, at least) as well and that's caused confusion in the past.

Reply:
Sure makes sense to me.